src/codegen/llvm/codegen_llvm_helper_visitor.cpp

```diff
@@ -450,6 +450,44 @@ void CodegenLLVMHelperVisitor::visit_function_block(ast::FunctionBlock& node) {
     create_function_for_node(node);
 }
 
+/// Create asr::Varname node with given a given variable name
+static ast::VarName* create_varnamae(const std::string& varname) {
```

Comment:
@pramodk nitpick: typo in function name:)

Reply:
Disadvantages of modern IDEs - once you make typo, rest is just autocomplete ! 😄 
fixes, thanks!